api-specificatie/common.yaml

```diff
@@ -150,9 +146,7 @@ components:
       description: Gewenste CRS van de coördinaten in de response.
       schema:
         description: 'CRS van de coördinaten in de response. epsg:28992 mapt op het RD New Nederlands coordinatenstelsel.'
```

Comment:
hier heb je nog een description naast de $ref staan. mag in deze versie OAS niet, en is niet nodig want een niveau hoger is ook description

Reply:
eventueel kan je het tweede deel van de description verplaatsen naar CrsEnum, zodat duidelijk is dat epsg:28992 mapt op het RD New Nederlands coordinatenstelsel

Reply:
description verplaatst